### ISSUE TITLE
Feature: Surface blocks placement according to the terrain angle

### ISSUE THREAD

**Issue by Darsenia** [PG85/OpenTerrainGenerator#448]
In World Painter tool there is one cool feature,that I always wanted to have in OTG: surface blocks can be placed on terrain slopes with certain angle.This is extremely useful thing,it helps, for example, to create realistic mountains with snow only on appropriate slopes(slopes with higher angles are too steep)and other stuff.I'm not sure if this could be done for the ingame worldgen without performance issues...or if it is even possible:)But if OTG would have such feature, even more fantastic landscapes would be possible!

**Comment by MCPitman** [comment]
Yeah it's a good idea and something I've been asking to be added for probably years now.

To be honest I'd be surprised to see it happen without more developer manpower coming on board. PG has enough to do being a one man army as it is.

**Comment by PG85** [comment]
https://streamable.com/v7hx09

Looks great, would be easy to add for base terrain, more difficult / performance-intensive for resources/objects/structures etc. Putting this in the backlog for now.
